README.md

```diff
@@ -161,8 +167,78 @@ code using [eslint](https://eslint.org/), check code formatting
 using [prettier](https://prettier.io/) and
 `npm audit` & `npm outdated` to run node package manager audit.
 
+## Deployment Instructions
+
+---
+
+**Time to deploy**: Approximately 10 minutes
+
+- [ ] Download the source code repo using below command and change directory
+
+```console
+git clone https://gitlab.aws.dev/aws-wwps-uk-proserve-edu/trusted-research-environment/opensource/secure-egress-frontend.git
```

Comment:
We probably want to change this to the TREEHOOSE repo?

Reply:
Done